shared/middleware/helpers/storage_apps.rb

```diff
@@ -347,10 +385,19 @@ def self.remix_ancestry(channel_id, depth: 1)
 
   def self.get_abuse(channel_id)
     _, storage_app_id = storage_decrypt_channel_id(channel_id)
-    project_info = PEGASUS_DB[:storage_apps].where(id: storage_app_id).first
+    project_info = StorageApps.table.where(id: storage_app_id).first
     project_info[:abuse_score]
   end
 
+  # Returns storage apps with id in ids
+  def self.get_by_ids(ids)
+    StorageApps.table.where(id: ids)
+  end
+
+  def self.table
```

Comment:
How much additional work is it to make this private?  (I'm guessing projects_list is going to biggest pain here, ok to defer until later but I think we'll want to make this private at some point.)

Reply:
Ideally in the future we'll be using an ActiveRecord model for queries and we won't need to explicitly reference the table